src/dualscreen.tsx

```diff
@@ -23,7 +23,7 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
   public secondaryKalturaPlayer: KalturaPlayerTypes.Player;
   private _player: KalturaPlayerTypes.Player;
   private _layout: Layout;
-  private _streamLayout: StreamLayout | null = null;
+  private _streamLayout: ExternalLayout | null = null;
```

Comment:
please update also property name from _streamLayout to externalLayoutChange or something